src/result_object/helpers/fail/index.js

```diff
@@ -1,7 +1,8 @@
 // @flow
 import PassableResponse from '../../index';
 
-export const WARN: string = 'warn';
+export const WARN: Severity = 'warn';
+export const FAIL: Severity = 'fail';
 
 type CountName = 'failCount' | 'warnCount';
```

Comment:
line 12: `const isFail: boolean = (severity !== WARN);` why not `const isFail: boolean = (severity === FAIL);` ?

Reply:
Adding WARN and FAIL exports, and checking explicit severity.
`const severity = [WARN, FAIL].includes(userSeverity) ? userSeverity : FAIL;`